### PR TITLE
Fixes bug that collator wasn't sending `Declare` message

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: node/network/bridge/src/lib.rs [hunk 1]
```diff
@@ -500,7 +500,7 @@ async fn send_message<M, I>(
 					.clone()
 			};
 
-			Ok(NetworkAction::WriteNotification(peer, peer_set, message))
+			Ok(NetworkAction::WriteNotification(peer, dbg!(peer_set), message))
```
Comment (montekki) [inline node/network/bridge/src/lib.rs:503]: `dbg`

File: node/network/bridge/src/lib.rs [hunk 2]
```diff
@@ -615,9 +615,9 @@ where
 
 			Action::SendCollationMessage(peers, msg) => send_message(
 					&mut network_service,
-					peers,
+					dbg!(peers),
 					PeerSet::Collation,
-					WireMessage::ProtocolMessage(msg),
+					dbg!(WireMessage::ProtocolMessage(msg)),
```
Comment (montekki) [inline node/network/bridge/src/lib.rs:620]: `dbg`

File: node/network/collator-protocol/src/collator_side.rs [hunk 3]
```diff
@@ -593,6 +597,7 @@ where
 			// it should be handled here.
 		}
 		PeerViewChange(peer_id, view) => {
+			println!("PEERVIEWCHANGE: {:?} {:?}", peer_id, view);
```
Comment (montekki) [inline node/network/collator-protocol/src/collator_side.rs:600]: `println`

**Review by montekki — APPROVED** [review]
LGTM minus debug prints

File: node/network/bridge/src/lib.rs (hunk 1 above)
Comment (bkchr) [reply node/network/bridge/src/lib.rs:503]: Fuck xD

Comment (bkchr) [inline node/network/bridge/src/lib.rs:503]:
```suggestion
			Ok(NetworkAction::WriteNotification(peer, peer_set, message))
```

File: node/network/bridge/src/lib.rs [hunk 4]
```diff
@@ -615,9 +615,9 @@ where
 
 			Action::SendCollationMessage(peers, msg) => send_message(
 					&mut network_service,
-					peers,
+					dbg!(peers),
```
Comment (bkchr) [inline node/network/bridge/src/lib.rs:618]:
```suggestion
					peers,
```

File: node/network/collator-protocol/src/collator_side.rs (hunk 3 above)
Comment (bkchr) [inline node/network/collator-protocol/src/collator_side.rs:600]:
```suggestion
```

File: node/network/bridge/src/lib.rs (hunk 2 above)
Comment (bkchr) [inline node/network/bridge/src/lib.rs:620]:
```suggestion
					WireMessage::ProtocolMessage(msg),
```

**Comment by montekki** [comment]
bot merge

**Pull request merged**